app/routers/ai_chat.py

```
@@ -0,0 +1,192 @@
+
+        return ChatResponse(
+            response=result["response"],
+            provider=result["provider"],
+            model=result["model"],
+            cost=result["cost"],
+            tokens_used=result["tokens_used"],
+            complexity=complexity.value,
+            fallback_used=False,
+        )
+
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)